Lumina/Lumina/UI/Extensions/Delegates/LuminaDelegate.swift

```diff
@@ -72,6 +72,10 @@ public protocol LuminaDelegate: class {
     ///
     /// - Parameter controller: the instance of Lumina that cancel was tapped on
     func dismissed(controller: LuminaViewController)
+    
+    /// Triggered whenever the image is tapped (outside of the buttons)
+    ///
+    func tapped(controller: LuminaViewController)
```

Comment:
I think, if we are going to add this functionality, we need to also pass back the coordinates of where the tap happened. It might not be valid for your specific use case here, but notifying not only the tap but where it happened might be valid data for another user.

Reply:
We can simply pass back a `CGPoint` here by passing in `recognizer.location(in: self.view)` as a parameter.